x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/data_provider.ts

```diff
@@ -13,8 +13,11 @@ export const IS_OPERATOR = ':';
 /** The `exists` operator in a KQL query */
 export const EXISTS_OPERATOR = ':*';
 
+/** The `is one of` faux operator in a KQL query */
+export const IS_ONE_OF_OPERATOR = 'includes';
+
 /** The operator applied to a field */
-export type QueryOperator = ':' | ':*';
+export type QueryOperator = ':' | ':*' | 'includes';
```

Comment:
this should be `export type QueryOperator = IS_OPERATOR | EXISTS_OPERATOR | IS_ONE_OF_OPERATOR;` i think

Reply:
Tried making that change and got:
`'IS_ONE_OF_OPERATOR' refers to a value, but is being used as a type here. Did you mean 'typeof IS_ONE_OF_OPERATOR'?`

changing it to be `typeof ...` ended up being equivalent to `type QueryOperator = ':' | ':*' | 'includes';`

Reply:
```typescript
export type QueryOperator = typeof IS_OPERATOR | typeof EXISTS_OPERATOR | typeof IS_ONE_OF_OPERATOR;
```

Works when you add a couple of `typeof`'s :)

Reply:
*(deleted my previous comment)*

FWIW: I think it still makes sense going the extra mile and adding the `typeof`s :)

Reply:
got it, will do, but can I ask why